syncano/models/manager.py

```diff
@@ -828,6 +828,26 @@ def _get_endpoint_properties(self):
         return self.model._meta.resolve_endpoint(self.endpoint, defaults), defaults
 
 
+class RuntimeChoices(object):
```

Comment:
@dancio @forgems So i removed this method and palce normal constants - what do ya think guys? I need to merge it and release LIB tomorrow to end first phase of parse task :)